sips/sip-019/sip-019-modify-coinbase-reward-per-block-and-halving-schedule.md

```diff
@@ -0,0 +1,74 @@
+# Preamble
+
+SIP Number: 019
+
+Title: Modify Coinbase Reward per Block & Halving Schedule
+
+Author: Xan Ditkoff (email: xan@daemontechnologies.co)
+
+Consideration: Technical, Governance, Economics
+
+Type: Standard
+
+Status: Draft
+
+Created: 16 May 2022
+
+License: CC0-1.0
+
+Sign-off: 
+
+# Abstract
+There are currently two ways the Stacks network rewards STX miners when they win a block: STX tokens as part of a “Coinbase reward” and STX tokens from transaction fees. The purpose of the Coinbase reward is to bootstrap and to provide an extra incentive for miners to secure the network in the early days, when transaction fees are not yet sufficient. 
+
+For the first 10,000 blocks of the Stacks mainnet launch the coinbase reward was 2000 STX compared with 1000 STX today. This period coincided with significant interest from users and miners. Historically, the year 2050 STX supply has been reduced in the past (from 5B to 2B to 1.8B). This proposal calls for a slight increase in the year 2050 supply, and increasing the current coinbase reward to 1600 STX until the next halving. 
+
+# License and Copyright
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at https://creativecommons.org/publicdomain/zero/1.0/ This SIP’s copyright is held by the Stacks Open Internet Foundation.
+
+# Introduction
+Open membership blockchain based networks need to provide an economic incentive for miners to process transactions and provide security. Coinbase rewards are typically used to bootstrap this economic incentive in the early days of an open source network when transaction fees alone will not provide a sufficient incentive. Currently, the coinbase reward for the Stacks network is 1000 STX per block, with a halving schedule as follows:
+
+<img width="659" alt="Screenshot 2022-05-16 at 2 35 35 PM" src="https://user-images.githubusercontent.com/31934541/168593660-d13311d9-2710-4202-bc9d-e41bf4d6c945.png">
+
+For the first ten thousand blocks of Stacks 2.0, however, the coinbase reward was inflated to 2000 STX per block, and resulted in much more favorable STX mining and Stacking conditions. Since the purpose of a coinbase reward is to bootstrap the network in the early days, we propose increasing the coinbase reward and subsequently modifying the halving schedule.
```

Comment:
There's not very much rationale here.  Can you expand on why you believe this particular tactic is the best approach to creating "much more favorable STX mining and Stacking conditions?"  Can you also expand what those conditions are, and why we believe they can be obtained?  Right now, Stacking gets ~7-8% APY (but is highly variable).  I'm skeptical we'd be able to do better than 7% sustainably, since that's what stock markets tend to get (after you adjust for inflation).

Reply:
Furthermore, the stacking rewards depends on the amount of stacked STX.